__tests__/app.test.js

```diff
@@ -69,6 +69,79 @@ afterAll(() => {
   });
 
 
+  describe("GET/api/users", () => {
```

Comment:
This test is not necessary as the endpoint should return all the users, please remove the test 🙂